### PR TITLE
178551916 cluster stats frontend

### PR DESCRIPTION
**Pull request by kbiala**
#### What's this PR do?
- Improves cluster stats frontend (only for developers team for now - no public link yet). This is work in progress - still needs to add validators to stats. We'll create public version eventually, and move all statistics from homepage here. Please ignore missing parts for now.

#### How should this be manually tested?
- Review in browser, link: .../cluster-stats/:network, or deploy to staging and review: https://stage.validators.app/cluster-stats/:network

#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/178551916

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/views/cluster_stats/index.html.erb [hunk 1]
```diff
@@ -1,16 +1,313 @@
...
+          <thead>
+          <tr>
+            <th>#</th>
+            <th>Validator</th>
+            <th>Stake</th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody>
+          <% @stats[:top_staked_validators].first(50).each.with_index do |v, index| %>
```
Comment (radek-polcode) [inline app/views/cluster_stats/index.html.erb:41]: We want to show only 50 validators? No pagination?

File: app/views/cluster_stats/index.html.erb [hunk 2]
```diff
@@ -1,16 +1,313 @@
...
+
+  <div class="col-12 col-md-6 mb-4">
+    <div class="card">
+      <div class="card-content">
+        <h3 class="card-heading mb-4">Skipped Slot&nbsp;&percnt;</h3>
+        <div class="row px-3 px-xl-5 mb-md-2">
+          <div class="stat col-5 my-auto">
+            <small class="text-muted">Median:</small>
+            <div class="h5 text-success">
+              <%= number_to_percentage @stats[:skipped_slots][:median].to_f*100, precision: 1 %>
```
Comment (radek-polcode) [inline app/views/cluster_stats/index.html.erb:261]: Add spaces near *

File: app/views/cluster_stats/index.html.erb [hunk 3]
```diff
@@ -1,16 +1,313 @@
...
+            </div>
+            <small class="text-muted">Average:</small>
+            <div class="h5 text-success">
+              <%= number_to_percentage @stats[:skipped_slots][:average].to_f*100, precision: 1 %>
```
Comment (radek-polcode) [inline app/views/cluster_stats/index.html.erb:265]: Add spaces near *

**Review by radek-polcode — COMMENTED** [review]
Add empty lines at the ond of the files
Consider splitting `app/views/cluster_stats/index.html.erb` to partials.

File: app/views/cluster_stats/index.html.erb (hunk 1 above)
Comment (kbiala) [reply app/views/cluster_stats/index.html.erb:41]: TBH I don't know yet how many records we'll want to display, maybe even fewer. So I left it like this for now.

**Pull request merged**